static/panes/editor.js

```diff
@@ -505,6 +489,54 @@ Editor.prototype.initButtons = function (state) {
     this.currentCursorPosition.hide();
 };
 
+Editor.prototype.handleCtrlS = function (event) {
+    event.preventDefault();
+    if (this.settings.enableCtrlStree && this.hub.hasTree()) {
+        var trees = this.hub.trees;
+        // todo: change when multiple trees are used
+        if (trees && trees.length > 0) {
+            trees[0].multifileService.includeByEditorId(this.id).then(_.bind(function () {
+                trees[0].refresh();
+            }, this));
+        }
+    } else {
+        console.log(typeof(this.settings.enableCtrlS));
+        if (this.settings.enableCtrlS === 'true') {
+            loadSave.setMinimalOptions(this.getSource(), this.currentLanguage);
+            if (!loadSave.onSaveToFile(this.id)) {
+                this.showLoadSaver();
+            }
+        } else if (this.settings.enableCtrlS === 'false') {
+            this.eventHub.emit('displaySharingPopover');
+        } else if (this.settings.enableCtrlS === '2') {
+            this.runFormatDocumentAction();
+        } else if (this.settings.enableCtrlS === '3') {
+            this.handleCtrlSDoNothing();
+        }
+    }
+};
+
+Editor.prototype.handleCtrlSDoNothing = function () {
+    if (this.nothingCtrlSTimes === undefined) {
+        this.nothingCtrlSTimes = 0;
+        this.nothingCtrlSSince = Date.now();
+    } else {
+        if (Date.now() - this.nothingCtrlSSince > 5000) {
+            this.nothingCtrlSTimes = undefined;
+        } else if (this.nothingCtrlSTimes === 4) {
+            var element = this.domRoot.find('.ctrlSNothing');
+            element.show(100);
```

Comment:
Shows a smily face after 5 hits in less that 5 seconds, because why not?
But open to removing it if not wanted

Reply:
I like it! Maybe we could also add some text hinting that they disabled Ctrl + S in the settings?

Reply:
lol :)